packages/file-upload/tests/site/index.html

```diff
@@ -0,0 +1,71 @@
+		</script>
+	<![endif]-->
+</head>
+<body>
+	<noscript>
+		<p role="alert">This website needs JavaScript to work properly.</p>
+	</noscript>
+
+	<script type="text/javascript" src="script.js"></script>
+
+	<a class="test-link" href="../">&larr; back to the module index</a>
+
+	<h1>Test: file-upload</h1>
+
+	<div class="au-file-upload js-upload-1">
+		<input type="file" id="au-file-upload-1" name="au-file-upload-1" class="au-file-upload__input">
+		<label title="upload file" for="au-file-upload-1" class="au-file-upload__label">
+			upload some file
+		</label>
+	</div>
+
+
+	<div class="au-file-upload js-upload-2">
```

Comment:
```suggestion
	<div class="au-file-upload" id="js-upload-2">
```

Reply:
Sure. It uses `querySelector`, you can pass an id selector `AU.fileUpload('#js-upload-2')`

Reply:
Yea I saw that, but the main reason is because we won't use the `.js-upload-2` selector on another element, so it's better to use `#js-upload-2` if that makes sense.